test/PerfTest/WebApiPerformance.Service/WebApiPerformance.Service.csproj

```diff
@@ -21,21 +21,52 @@
     <IISExpressWindowsAuthentication />
     <IISExpressUseClassicPipelineMode />
     <UseGlobalApplicationHostFile />
+    <TestType Condition=" '$(TestType)' == '' ">Private</TestType>
   </PropertyGroup>
+  <Choose>
+    <When Condition=" '$(TestType)'=='Private' ">
+      <ItemGroup>
+        <Reference Include="Microsoft.OData.Core, Version=7.2.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+          <HintPath>..\..\..\sln\packages\Microsoft.OData.Core.7.2.0\lib\portable-net45+win8+wpa81\Microsoft.OData.Core.dll</HintPath>
+          <Private>True</Private>
+        </Reference>
+        <Reference Include="Microsoft.OData.Edm, Version=7.2.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+          <HintPath>..\..\..\sln\packages\Microsoft.OData.Edm.7.2.0\lib\portable-net45+win8+wpa81\Microsoft.OData.Edm.dll</HintPath>
+          <Private>True</Private>
+        </Reference>
+        <Reference Include="Microsoft.Spatial, Version=7.2.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+          <HintPath>..\..\..\sln\packages\Microsoft.Spatial.7.2.0\lib\portable-net45+win8+wpa81\Microsoft.Spatial.dll</HintPath>
+          <Private>True</Private>
+        </Reference>
+        <ProjectReference Include="..\..\..\src\System.Web.OData\System.Web.OData.csproj">
+          <Project>{d23e28f1-ccd0-43e0-8c0d-36731ec91318}</Project>
+          <Name>System.Web.OData</Name>
+        </ProjectReference>
+      </ItemGroup>
+    </When>
+    <When Condition=" '$(TestType)'=='Official' ">
+      <ItemGroup>
+        <Reference Include="Microsoft.OData.Core">
+          <HintPath>..\..\..\sln\packages\Microsoft.OData.Core.$(ODataLibVersion).0\lib\portable-net45+win8+wpa81\Microsoft.OData.Core.dll</HintPath>
```

Comment:
1. For automatically build, It's input from Build Definition 
2. For manually, it's input by command line.